sapi/litespeed/lsapi_main.stub.php

```diff
@@ -2,19 +2,40 @@
 
 /** @generate-class-entries */
 
+/**
+ * @return array<string, string>
+ * @alias apache_request_headers
+ */
 function litespeed_request_headers(): array {}
 
-/** @alias litespeed_request_headers */
+/**
+ * @return array<string, string>
+ * @alias apache_request_headers
+ */
 function getallheaders(): array {}
 
-/** @alias litespeed_request_headers */
+/**
+ * @return array<string, string>
+ * @refcount 1
+ */
 function apache_request_headers(): array {}
 
+/**
+ * @return array<string, string>|false
+ * @refcount 1
+ */
 function litespeed_response_headers(): array|false {}
 
-/** @alias litespeed_response_headers */
+/**
+ * @return array<string, string>|false
+ * @alias litespeed_response_headers
+ */
 function apache_response_headers(): array|false {}
```

Comment:
Unlike other SAPIs, litespeed makes `apache_request_headers()` and `apache_response_headers()` an alias of its own implementations. The former one can be safely changed (so that `litespeed_request_headers()` becomes the alias of `apache_request_headers()`), however the latter one cannot, since the return type is different (`array|false` vs `array`). Can we somehow remove the `false` return? Or what could we do in order to add `apache_response_headers()` to the optimizer func info?